Cargo.toml

```diff
@@ -119,14 +118,14 @@ proc-macro2 = "1.0"
 quote = "1.0"
 rand = "0.8.5"
 regex = "1.7"
+reqwest = { version = "0.11.22", features = ["rustls-tls"] }
```

Comment:
I opted to keep `rustls` as an option, although I'm not 100% sure as to why.

Reply:
we use `rustls` because it's easier to target `musl` - the other option would be `native-tls`, which is a bit of a pain to set up


This is related to one of the reasons CI is failing - `reqwest`, by default, enables the `default-tls` which tries to use the `native-tls`, so in addition to `features = ["rustls-tls"]` you also need to disable the default feature:

```diff
- reqwest = { version = "0.11.22", features = ["rustls-tls"] }
+ reqwest = { version = "0.11.22", default-features = false, features = ["rustls-tls"] }
```